projects/shared-modules/mention/components/entity-profile/entity-profile.component.scss

```diff
@@ -0,0 +1,105 @@
+@import '/projects/commudle-admin/src/assets/styles/variables';
+
+.user {
+  padding: 0.5rem;
+  display: flex;
+  align-items: flex-start;
+  position: relative;
+
+  &.small {
+    align-items: center;
+  }
+
+  .online-status {
+    position: absolute;
+    top: 6px;
+    left: 3px;
+    width: 10px;
+    height: 10px;
+    border-radius: 50%;
+
+    &.online {
```

Comment:
Are these styles being used in the corresponding HTML file? If not, let's remove it